model/common/tests/grid_tests/test_topography.py

```diff
@@ -0,0 +1,85 @@
+# ICON4Py - ICON inspired code in Python and GT4Py
+#
+# Copyright (c) 2022-2024, ETH Zurich and MeteoSwiss
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import pytest
+
+from icon4py.model.common import dimension as dims, type_alias as ta
+from icon4py.model.common.grid import geometry, topography as topo
+from icon4py.model.common.test_utils import datatest_utils as dt_utils, helpers, reference_funcs
+
+
+@pytest.mark.datatest
+@pytest.mark.parametrize(
+    "experiment",
+    [
+        (dt_utils.GAUSS3D_EXPERIMENT),
+    ],
+)
+def test_topography_smoothing_with_serialized_data(
+    icon_grid,
+    grid_savepoint,
+    interpolation_savepoint,
+    external_parameters_savepoint,
+    backend,
+    experiment,
+):
+    cell_geometry: geometry.CellParams = grid_savepoint.construct_cell_geometry()
+    geofac_n2s = interpolation_savepoint.geofac_n2s()
+
+    num_iterations = 25
+    topography = external_parameters_savepoint.topo_c()
+    topography_smoothed_verif_np = external_parameters_savepoint.topo_smt_c().ndarray
+
+    topography_smoothed = topo.compute_smooth_topo(
+        topography=topography,
+        grid=icon_grid,
+        cell_areas=cell_geometry.area,
+        geofac_n2s=geofac_n2s,
+        backend=backend,
+        num_iterations=num_iterations,
+    )
+
+    assert helpers.dallclose(
+        topography_smoothed_verif_np, topography_smoothed.ndarray, atol=1.0e-14
+    )
+
+
+@pytest.mark.datatest
+def test_topography_smoothing_with_numpy(
```

Comment:
What are you trying to test with this test? When would that test fail (and the one above would not)? Could you test that what you want to test without any serialized data? Or is it not useful at all?

(The question what they actually test, is one I have for all numpy tests for stencils (aka `StencilTest`) that we have. They have limited use IMHO but were handy while porting stencils and might be handy for benchmarking..